lib/stat_tracker.rb

```diff
@@ -86,4 +86,19 @@ def average_goals_per_game
     end
     average = (per_game_average.sum / per_game_average.size).round(2)
   end
+
```

Comment:
Glad you got this figured out! It will come in handy for the best offense method we're about to take on, too.